source/dev/idea_list.md

```diff
@@ -0,0 +1,165 @@
+
+#### Project 1a. - Download Manager
+The goal is to encapsulate the existing code for downloading external resources into a single library and Command Line Interface (CLI).
+
+The result of this project will be a single CLI tool that we can use to download external resources, either manually, or by integrating into an orchestration/workflow system such as Apache Airflow.
+
+- Write a Python library and CLI that downloads external data sources
+- The tool should be well documented with a focus on code quality
+
+**Requirement:** Python
+
+**Level:** Easy
+
+**Mentor:**
+[Todd](https://github.com/ToddG)
+
+#### Project 1b. - Validation Manager
+The goal of this project is to encapsulate the existing validation and cleaning of external data resources into a single library and CLI.
+
+The result of this project will be a single CLI tool that we can use to validate external resources, either manually, or by integrating into an orchestration/workflow system such as Apache Airflow.
+
+- Write a command-line tool that verifies previously downloaded external data resources
+- The tool ensures that the data is correct (format, data types, value domains, missing values, etc.), and either cleans the input data or responds with an error
+- The tool should be well documented with a focus on code quality
+
+**Requirement:** Python
+
+**Level:** Easy
+
+**Mentor:**
+[Todd](https://github.com/ToddG)
+
+#### Project 1c. - Ingestion Pipeline
+The goal of this project is to automate the download, validation, and archival of external resources using industry standard components such as Apache Airflow, InfluxDB, Grafana, Blob Stores, etc.
+
+Note that this project will eventually incorporate the work of the Validation Manager and Download Manager projects. However, each of these projects can proceed in parallel.
+
+- Write Apache Airflow workflows that periodically
+    - Query external data sources for availability
+    - Download external data sources
+    - Validate and clean the downloaded resources
+    - Archive downloaded resources to a blob store
+- Workflows are instrumented with telemetry
+- Telemetry are aggregated in InfluxDB
+- Metrics are displayed in a Grafana dashboard
+- If a data source is down, we throw an alarm
+
+The workflows should be well documented with a focus on code quality, and use best-of-breed current technology stacks (Apache Airflow, Kubernetes, Helm, Docker, Python3, Grafana, etc.)
+
+**Requirements:**
+Docker, Python, Grafana, PAAS, Apache Airflow
+
+**Level:**
+Medium to Hard
+
+**Mentor:**
+[Todd](https://github.com/ToddG)
+[Jon](https://github.com/jon-hagg)
+
+### Project 2. - Test system for end-to-end testing of the simulation framework
+Use [existing power systems test grids](http://labs.ece.uw.edu/pstca/) in order to to benchmark our simulation framework
```

Comment:
Repeated "to"

Reply:
BTW found these in the logs from the [style check](https://github.com/Breakthrough-Energy/docs/pull/48/checks?check_run_id=1937181400).

Reply:
I should have looked there.

Reply:
I keep forgetting we had it